CHANGELOG.md

```diff
@@ -178,9 +213,6 @@ Your Cumulus Message Adapter version should be pinned to `v1.0.13` or lower in y
 
 - **CUMULUS-1448** Refactor workflows that are mutating cumulus_meta to utilize meta field
 
-- **CUMULUS-1375**
-  - Migrate Cumulus from deprecated Elasticsearch JS client to new, supported one in `@cumulus/api`
```

Comment:
It doesn't seem like this was moved/duplicated anywhere, was this delete intentional?

Reply:
Misread the change, nevermind!